runtime/build.gradle.kts

```diff
@@ -167,11 +170,16 @@ targetList.forEach { targetName ->
         includeRuntime()
     }
 
+    // TODO: This "all tests" tasks should be provided by `CompileToBitcodeExtension`
     tasks.register("${targetName}RuntimeTests") {
-        dependsOn("${targetName}StdAllocRuntimeTests")
-        dependsOn("${targetName}MimallocRuntimeTests")
-        dependsOn("${targetName}ExperimentalMMStdAllocRuntimeTests")
-        dependsOn("${targetName}ExperimentalMMMimallocRuntimeTests")
+        val platformManager = project.rootProject.findProperty("platformManager") as PlatformManager
+        val target = platformManager.targetByName(targetName)
+        target.supportedSanitizers().forEach { sanitizer ->
+            dependsOn("${targetName}StdAllocRuntimeTests${CompileToBitcodeExtension.suffixForSanitizer(sanitizer)}")
+            dependsOn("${targetName}MimallocRuntimeTests${CompileToBitcodeExtension.suffixForSanitizer(sanitizer)}")
+            dependsOn("${targetName}ExperimentalMMStdAllocRuntimeTests${CompileToBitcodeExtension.suffixForSanitizer(sanitizer)}")
+            dependsOn("${targetName}ExperimentalMMMimallocRuntimeTests${CompileToBitcodeExtension.suffixForSanitizer(sanitizer)}")
+        }
```

Comment:
This part is kinda too intrusive. Maybe make `createTestTasks` return list of tasks and use the results here?